docs/data-model.md

```diff
@@ -110,26 +102,26 @@ Additionally, Providers will need to be onboarded to use the School Placements s
 
 ### Schools
 
-An 'onboarded' School will have a record in the `schools` table (the `School` entity in our ERD).
+All schools from the gias import will have a record in the `schools` table (the `School` entity in our ERD).
```

Comment:
Let's try to stylise it as GIAS wherever possible. That's what's used on [the service's about page](https://get-information-schools.service.gov.uk/about).

```suggestion
All schools from the GIAS import will have a record in the `schools` table (the `School` entity in our ERD).
```